continuserv: Server to live serve local updates to the spec

illicitonion opened the pull request:
The go isn't wonderful, but I hacked this together for my own benefit, and figure other people may want to use it

Kegsay commented:
Install instructions would be great (e.g. min Go version and basic install steps) to include in a `README` in the `continuserv` directory. FWIW, I had to `go get` to get `fsnotify` then `go run main.go`.

It also didn't seem to pick up changes in the `/specification` folder when I tried - though it did on the YAML schemas in `/api`. The directories used to generate the spec are:
```
/api
/event-schemas
/scripts (specifically the .css files)
/specification
/templating
```
If you could watch these directories that would encompass any and all changes to the spec, not just YAML stuff.

illicitonion commented:
Added README.

I also added a logging line, so that it will log when it detects an update. It should be finding any changes in the git repo, and I certainly could trigger rebuilds with changes to /specification - if you can reproduce with the logging, let me know :)

Kegsay commented:
LGTM
